src/parameter_input.cpp

```diff
@@ -109,6 +109,11 @@ InputBlock::~InputBlock() {
   }
 }
 
+void toLower(std::string &name) {
+  std::transform(name.begin(), name.end(), name.begin(),
+                 [](unsigned char c) { return std::tolower(c); });
+}
```

Comment:
I can't believe there's no built-in `to::lower` for an entire string. But there does not seem to be. That said, you don't need the lambda.
```C++
void toLower(std::string &name) {
  std::transform(name.begin(), name.end(), name.begin(), std::tolower);
}
```